src/server/repositories/product.repository.ts

```diff
@@ -0,0 +1,70 @@
+import Product from "../models/Product.models";
+
+const getOneById = async (id: number, userId: number) => {
+  const product = await Product.findAll({
+    where: {
+      id,
+      user_id: userId,
+    },
+  });
+  if (product[0]) {
+    return product[0];
+  }
+  return null;
+};
+const getAll = async (userId: number) => {
+  const products = await Product.findAll({
+    where: {
+      user_id: userId,
+    },
+    order: [["id", "DESC"]],
+  });
+  return products;
+};
+
+const findByIdAndUpdate = async (
+  id: number,
+  title: string,
+  price: number,
+  userId: number
+) => {
+  const countUpdated = await Product.update(
+    { title, price, userId },
+    {
+      where: {
+        id,
+        user_id: userId,
+      },
+    }
+  );
+
+  if (countUpdated[0] > 0) {
```

Comment:
you can call `return countUpdated[0] > 0`as you did in `findByIdAndDelete` method